.github/workflows/main_win.yaml

```diff
@@ -138,7 +138,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
         with:
-          fetch-depth: 1
+          fetch-depth: 0
```

Comment:
Can we add a comment here explaining why we want fetch-depth pinned to 0?  This will help avoid someone in the future "optimize" workflow by limiting number of commits being fetched.

Reply:
Alternatively we can just do it like this: https://gist.github.com/0penBrain/7be59a48aba778c955d992aa69e524c5

Reply:
Actually this would be slightly difficult due to it being executed in cobalt build code